Tutorial company level experiments (#5816)
PostHog/posthog.com: contents/blog/running-group-targeted-ab-tests.md

## Combining user-targeted and group-targeted experiments to reduce risk

To improve the quality of insights from group-targeted experiments and address the risks mentioned above, you may want to consider running an experiment with your change at a user-level before running it at a group-level. This can help you in the following ways:

### 1. Save time for changes that don't have an impact

As mentioned, lower statistical power in group-targeted experiments means you need to run your experiments longer. However, a change that doesn't impact behavior at a user-level won't, in all likelihood, impact behavior at a group-level either.

To rule this out, consider running a user-targeted experiment first in order to see if there is any impact on individual user behavior (where it should be quicker to gain results from because of higher statistical power) before running a group-targeted experiment.

### 2. Gain user-level insights and identify randomization risks

> By running a user-targeted experiment first, you'll gain insight into how individual users interact with your change and what characteristics they may possess. This will then give you an understanding of what results you can expect from a group-targeted experiment.
> 
> This can also help you identify randomization risks. For example, let's say a B2B SaaS app runs a user-targeted experiment and notices that users in large companies are more likely to interact with their new feature. In this case, when they run a group-targeted experiment, they can ensure to filter out small and medium-sized companies in their experiment in order to obtain more accurate results.

Reviewer: ```suggestion
Running a user-targeted experiment first gives you insight into how individual users interact with your change and what properties they possess. This will then give you an understanding of what results you can expect from a group-targeted experiment.

This can also help you identify randomization risks. For example, let's say a B2B SaaS app runs a user-targeted experiment, and notices that users in large companies are more likely to interact with their new feature. In this case, when they run a group-targeted experiment, they can ensure to filter out small and medium-sized companies in their experiment in order to obtain more accurate results.
```

## How to set up group-targeted experiments in PostHog

First, ensure you have [set up a group](docs/product-analytics/group-analytics#getting-started) in PostHog. Next, go to your PostHog dashboard and create a [new experiment](docs/experiments/manual#creating-an-experiment).

In the New Experiment screen, select the Participant type. By default, it will show `Persons` (i.e. a user-level experiment). Click on the drop-down and select your new group.

![Screenshot of setting the participant type in an experiment](../images/blog/running-group-targeted-ab-tests/participant-type.png)

Next, set your experiment goal and secondary metrics as usual. 

A handy feature especially useful for group-level experiments is the Minimum Acceptable Improvement calculator at the bottom of the page. Since group-level experiments have lower statistical power, this will recommend a time for how long you should run your experiment in order to see results.

![Screenshot of the Minimum Acceptable Improvement calculator](../images/blog/running-group-targeted-ab-tests/minimum-acceptable-improvement.png)

> 

Reviewer: This article needs a summary to wrap it up – basically imagine a TL;DR. Something like:

"To summarize, remember these x points when running group-targeted experiments:

1. XYZ
2. XYZ
3. XYZ

To learn more about running product experiments, read our guides to `links`.

---
title: When and how to run group-targeted A/B tests
sidebar: Docs
showTitle: true
author: ['lior-neu-ner']
date: 2023-04-24

> featuredImage: ../images/blog/running-group-targeted-ab-tests/featured-image.png

Reviewer 2: White might be a little jarring as a background colour, maybe try the same color as the tutorial backgrounds? (E5E7E0)

Author: think I'll just ask Lottie for an image. Fits in better with the other blogs as well

Reviewer: Assuming this stays in the tutorials section, it should retain the same style. I'm sure there's one we can reuse.

Reviewer: Ah, I see it's moved to the blog already. That being the case, @Lior539 you need to update the frontmatter to the blog version. You can grab it from any existing blog post.

1. B2B SaaS apps, in order to test how a change will affect how an entire company uses your product.

   - For example, suppose Asana wants to test a new AI feature that automatically assigns tasks based on their urgency and importance. By conducting a group-targeted experiment, they can measure the impact on project completion rates across the entire company. 

2. Products with network effects, since a change for one user will likely affect how other users interact with the product.

   - For example, suppose Slack wants to improve the usage of a new video calling feature. Improving the feature's discoverability for a single user will increase their own usage with it, but since they use it with their coworkers, their coworkers will also discover it.

## How does running a group-targeted experiment differ from a user-targeted one?

There are three key differences to take into account when running a group-targeted experiment versus a user-level one:

### 1. Less statiscal power

Since you treat a group of users as a single data point in group-targeted experiments, you have less statistical power. For example, Slack has 20 million users, but only 600,000 companies using it.

> In practice, this means that you'll usually have to run group-targeted experiments longer than you would user-targeted experiments. It also means that your results may be more susceptible to noise.

Reviewer 2: "More susceptible to noise" is a little unclear in this context, could use with some more clarification. Maybe another sentence on why noise is bad? It makes it less clear the success of an experiment?

For example, say a B2B SaaS tool wants users to invite coworkers to use the app. They add a big "Invite Coworker" button and test it in a company-targeted experiment. The results show an increase in invites sent, but it's hard to know how individual users used the button. For example, what are the roles or skill sets of team members who are most likely to interact with the button and send invites?

## Combining user-targeted and group-targeted experiments to reduce risk

To improve the quality of insights from group-targeted experiments and address the risks mentioned above, you may want to consider running an experiment with your change at a user-level before running it at a group-level. This can help you in the following ways:

### 1. Save time for changes that don't have an impact. 

As mentioned, lower statistical power in group-targeted experiments means you need to run your experiments longer. However, a change that does not impact behavior at a user-level will not likely translate into impact at a group-level. So you may waste time waiting for results from a group-level test that has no impact.

> In this case, it may be worth running a user-targeted experiment first in order to see if there is any impact on individual user behavior (which should be quicker to gain insights from because of higher statistical power) before running a group-targeted experiment.

Reviewer 2: ```suggestion
In this case, it may be worth running a user-targeted experiment first in order to see if there is any impact on individual user behavior (where it should be quicker to gain results from because of higher statistical power) before running a group-targeted experiment.
```
